packages/flutter/lib/src/cupertino/route.dart

```diff
@@ -999,6 +999,11 @@ class _CupertinoEdgeShadowPainter extends BoxPainter {
 /// The `routeSettings` argument is used to provide [RouteSettings] to the
 /// created Route.
 ///
+/// [DisplayFeatureSubScreen] is used for displaying this route on only one
+/// sub-screen.
+///
+/// {@macro flutter.widgets.DisplayFeatureSubScreen}
```

Comment:
You can put a macro in another template here and reuse. :)

```suggestion
// make this whole bit another template
/// [DisplayFeatureSubScreen] can be used for displaying this route on only one
/// sub-screen.
///
/// {@macro flutter.widgets.DisplayFeatureSubScreen}
```

Also, what if I have a fully open, non-hinged screen and I want it in the middle?
Some sample code may be nice too to illustrate how this works.

Reply:
Tried putting a macro inside a template and all usages seemed to fail with `Found unresolved dartdoc directives in ...`. Does using a macro inside a template require doing anything extra?

Reply:
I've seen that happen if say the `template` documentation includes a `[reference]` that does not exist in the class where you place the `macro`, but it looks like you worked this out. :)